spec/index.bs

```diff
@@ -5096,20 +5106,25 @@ dictionary GPUExternalTextureDescriptor
 
                     1. Let |result| be a new {{GPUExternalTexture}} object wrapping |data|.
 
-                1. [$queue an automatic expiry task$] with device |this| and the following steps:
+                1. If |source| is an {{HTMLVideoElement}},
+                    [$queue an automatic expiry task$] with device |this| and the following steps:
 
                     <div data-timeline=content>
                         1. Set |result|.{{GPUExternalTexture/[[expired]]}} to `true`,
                             releasing ownership of the underlying resource.
                     </div>
 
                     Note:
-                    An external video texture should be imported in the same task that samples the texture
+                    An {{HTMLVideoElement}} should be imported in the same task that samples the texture
                     (which should generally be scheduled using `requestVideoFrameCallback` or
                     {{AnimationFrameProvider/requestAnimationFrame()}} depending on the application).
                     Otherwise, a texture could get destroyed by these steps before the
                     application is finished using it.
+                1. If |source| is a {{VideoFrame}}, then when |source| is closed, run the following steps:
```

Comment:
nit: Is there a definition of "closed" for a `VideoFrame` we can link to, here and above?

Reply:
Unfortunately not really, not at the moment. I didn't link to this algorithm because it's noexport:
https://www.w3.org/TR/webcodecs/#close-videoframe
But maybe I should just link to it anyway; we do this for a bunch of unexported HTML spec things that were important to link to too.